### PR TITLE
added ivy's details

### PR DESCRIPTION
**Pull request by Ivy8127**
Added my details to the page
I also forgot to create a branch and got lazy ...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -639,17 +639,17 @@ <h4>William Anderson</h4>
...
-                <img src="assets/img/team/team-4.jpg" class="img-fluid" alt="">
+                <img src="assets/img/team/team-5.png" class="img-fluid" alt="Ivy Mboya">
                 <div class="social">
-                  <a href=""><i class="bi bi-twitter"></i></a>
-                  <a href=""><i class="bi bi-facebook"></i></a>
-                  <a href=""><i class="bi bi-instagram"></i></a>
-                  <a href=""><i class="bi bi-linkedin"></i></a>
+                  <a href="https://twitter.com/aaaives"><i class="bi bi-twitter"></i></a>
+                  <a href="#"><i class="bi bi-facebook"></i></a>
+                  <a href="#"><i class="bi bi-instagram"></i></a>
```
Comment (RuthNjeri) [inline index.html]: If the links are not present, remove the tags.
```suggestion

```

File: index.html [hunk 2]
```diff
@@ -639,17 +639,17 @@ <h4>William Anderson</h4>
...
+                  <a href="https://www.linkedin.com/in/ivy-mboya/"><i class="bi bi-linkedin"></i></a>
                 </div>
               </div>
               <div class="member-info">
-                <h4>Amanda Jepson</h4>
-                <span>Accountant</span>
+                <h4>Ivy Mboya</h4>
+                <span>Software Engineer</span>
```
Comment (RuthNjeri) [inline index.html]: This code should be after line 662 below, in the row div

**Review by RuthNjeri — CHANGES_REQUESTED** [review]
Good first attempt @Ivy8127! The image should be 600 by 600 pixels. Also, is there an image named `team-5` in the assets? I cannot see it in the file changes.

**Comment by RuthNjeri** [comment]
Hi @Ivy8127, did you look at the comments above?

**Comment by Ivy8127** [comment]
> Hi @Ivy8127, did you look at the comments above?

Hi, @RuthNjeri  I've made corrections to the comments you made. About the width and height, the width and height style attributes aren't changing to 600 px though when I locally view the image. Is it the bootstrap? How do I change that?

**Comment by RuthNjeri** [comment]
Hi @Ivy8127, can you share a screenshot of what's happening on your end?

**Comment by Ivy8127** [comment]
> Hi @Ivy8127, can you share a screenshot of what's happening on your end?

@RuthNjeri  Sure : [
image
](url)

**Comment by RuthNjeri** [comment]
Hi Ivy, what would happen if you use inline styling and add the ! important rule?

**Comment by RuthNjeri** [comment]
Feedback implementation took too long. Closing the PR.

**Pull request closed without merging**